monty.qmd

```diff
@@ -7,15 +7,94 @@ source("common.R")
 
 Before showing how to fit odin models to data, we'll focus for a bit on monty itself.  If you are anxious to get on and fit the model from @sec-data-filter, you might skip ahead to @sec-inference, where we resume this.
 
-There are a bunch of things we want to cover in this chapter so likely it will be split into a few:
+The `monty` R package is designed for modular work with statistical distributions, leveraging Monte Carlo methods for sampling. It enables the construction of increasingly complex models through four approaches:
 
-* defining a model in monty, both easily with `monty_function` and less easily with `monty_model`
-* fitting models with MCMC
-* writing priors with the `monty_dsl`
-* using observers to follow along with the model (if we can come up with a good example)
-* working with nested models (once we write the code that would allow this)
-* running MCMC chains in parallel
-* exporting chains to work with other packages
-* different samplers
+1. Simple R code,
+2. A dedicated domain-specific language (DSL),
+3. Integration of `odin` models,
+4. Composition of two `monty` models.
 
-Hmm, there really is quite a bit of ground to cover here!
+The most basic method to define a `monty` model is through a simple R function, as shown in this chapter. More advanced examples are covered in the @sec-monty-models in particular in the context of Bayesian statistics. @sec-monty-dsl introduces the monty DSL for more versatile model building using a small probabilistic DSL similar to BUGs. @sec-monty-combine explains how to compose new monty models by combining two existing ones. The last part of this book starting from @sec-inference demonstrates how to incorporate `odin` models into `monty` workflows.
```

Comment:
Referring to sec-monty-combine here, doesn't seem to exist currently

Reply:
Removed reference.